src/main/java/analyzer/solutions/OptimalFormat.java

```diff
@@ -0,0 +1,62 @@
+package analyzer.solutions;
+
+import com.github.javaparser.ast.expr.*;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.Node;
+import static com.github.javaparser.JavaParser.*;
+import java.io.File;
+import com.github.javaparser.JavaParser;
+import java.util.List;
+import com.github.javaparser.ast.visitor.VoidVisitorAdapter;
+import com.github.javaparser.ast.stmt.IfStmt;
+import com.github.javaparser.ast.stmt.ReturnStmt;
+import com.github.javaparser.ast.visitor.TreeVisitor;
+import java.util.ArrayList;
+
+
+public class OptimalFormat {
+
+    public static CompilationUnit getContent(String flname) throws Exception{
```

Comment:
I'd recommend only having this throw the specific exceptions that it might need to throw. It appears there are only two that we need to be concerned about: ParseProblemException and
FileNotFoundException (https://www.javadoc.io/doc/com.github.javaparser/javaparser-core/3.14.5).

We may even want to consider a try-catch here and handling the two cases ourselves.